cypress/integration/finance/ledgers/ledgers.create.spec.js

```diff
@@ -0,0 +1,25 @@
+import Ledgers from '../../../support/fragments/finance/ledgers/Ledgers';
+
+import {
+  Button
+} from '../../../../interactors';
+
+describe('ui-finance: Ledger list search and filters', () => {
+  before(() => {
+    cy.login('diku_admin', 'admin')
+    .then(() => {
+        cy.visit('/finance/ledger');
+    });
+  });
+
+  it('should create new ledger if mandatory fields are filled', function () {
+    // create and verify Ledger via UI
+    Ledgers.createDefaultLedger();
+    // delete created Ledger
+    cy.do([
+            Button('Actions').click(),
```

Comment:
May be better use any string names like Actions, Delete or something else inside fragments? 
Because in future it can be renamed